pkg/lvm/lvm_util.go

```diff
@@ -89,6 +112,7 @@ func CreateVolume(vol *apis.LVMVolume) error {
 	out, err := cmd.CombinedOutput()
 
 	if err != nil {
+		err = newExecError(out, err)
```

Comment:
should we use newExecError in the previous return at line 103?

Reply:
It's not required. As the name suggests, newExecError just wraps only the error returned by `exec` pkg after running the cmd. Wrapping other errors will not provide any extra information since output will be empty.